style.css

```diff
@@ -0,0 +1,3 @@
+h1 {
+  color: brown;
+}
```

Comment:
- [ ] Please adhere to the `project requirement` because it will help you in future for example you got work from a client and the client sent the requirements and you implemented something `against` his/her requirement then the service you provided to your client can be considered as the *Bad User Experience* because adding your creativity as the developer is a very good thing but there will be no guarantee that your client will like it. I hope you understand this example.
Kindly look at the image attached below


![Screenshot from 2023-04-26 21-33-36](https://user-images.githubusercontent.com/66476329/234634620-315d1ba8-d4d3-4d8e-b2bc-e53ae768501f.png)

- So please make sure that you need to add a *class* for your `h1` tag inside the `index.html` file.

**NOTE: Always try to add your creativity after fulfilling the project requirement.**

Reply:
@skyv26 This problem solved